### DISCUSSION TITLE
Spare PCB boards for NL

### DISCUSSION THREAD

**Discussion by hjkoster400d** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#302] [General]
Recently ordered this board at JLCPCB, minimum quantity is 5, so i have some unused boards laying around. 
When you have interest is such a board, drop me a note and i will send it to you for 1 euro + transport costs (envelope)

**Comment by VincentCorniglion** [comment]
Hello, I would be interested. Do you have a picture of the PCB to make sure it's the one I need?

**Reply by hjkoster400d** [reply]
I ordered them last week, so expect them to be delivered somewhere in the next two weeks.

This is a screenshot of the board that i ordered
![image](https://user-images.githubusercontent.com/7768158/174779056-87db7570-c502-483a-9bdb-3e1da049b521.png)

**Reply by hjkoster400d** [reply]
![image](https://user-images.githubusercontent.com/7768158/174779271-d813bb39-cbe2-4a76-869d-9b744d4ae4b1.png)

**Reply by hjkoster400d** [reply]
PCB's arrived today :D

**Reply by Remigiusz2020** [reply]
HI,
Do you have still one left?

BR
Remi

**Comment by eht584** [comment]
Hello, I would be interested, it is possible paylpal and sent to Germany?
Eduard

**Reply by hjkoster400d** [reply]
Yeah sure, pcb's arrived today. Send me a PM with the address, and i will calculate the shippingcosts

**Reply by hjkoster400d** [reply]
PCB + protective foil + envelop = about 22gr. That means 2 stamps are required. Total shipping cost will be 3,10 euro. So total would then be 4,10 euro

**Comment by sponte** [comment]
Hello @hjkoster400d, would you ship it to the UK please?

**Reply by hjkoster400d** [reply]
Hi, yeah sure. Not sure about the shipping costs though, let me check...

**Reply by hjkoster400d** [reply]
PCB + protective foil + envelop = about 22gr. That means 2 stamps are required. Total shipping cost will be 3,10 euro. So total would then be 4,10 euro. In pounds that would be 3.5314 GBP ;)

**Reply by sponte** [reply]
Thanks so much for checking. Paypal good?

**Reply by hjkoster400d** [reply]
Just send you a payment request, let me know the recipient address and i'll make sure it's on it's way.

**Reply by sponte** [reply]
Brilliant, thanks @hjkoster400d . I sent the money and emailed you my address

**Reply by Deefjes** [reply]
Hoi Hjkoster.  

Do you have one board for me? 
I’m from NL. Thanks!

**Reply by hjkoster400d** [reply]
Yeah sure, send me the address and i wil send it to you.

**Comment by Deefjes** [comment]
Hi Rik,

Mijn adresgegevens:
Dave Leijsten
Maassingel 88
5751vp Deurne

Waar kan ik geld naar overmaken?
En hoeveel was het ook alweer?


Met vriendelijke groet,
Dave Leijsten

Op 10 jul. 2022 om 14:00 heeft Rik Koster ***@***.***> het volgende geschreven:

﻿

Yeah sure, send me the address and i wil send it to you.

**Comment by hjkoster400d** [comment]
Heey,

Verzending is 3,10 (bordje + noppenfolie + brief is net iets boven de
20gr helaas).
Voor het bordje is 1 euro (symbolisch) prima, dus totaal 4,10 euro.

Mag naar mijn Paypal, maar ik kan binnen nederland ook een tikkie sturen
als je dat liever hebt.

Ik zal de PCB vast netjes inpakken.

Gr Rik


Op zo 10 jul. 2022 om 15:10 schreef Deefjes ***@***.***>:

**Reply by Deefjes** [reply]
Thanks! Just transferred ze monnie!

**Comment by buteo72** [comment]
Hello,

Do you have still one left?
Possible to send it to Nuenen?

Thanks,
Christian

**Reply by hjkoster400d** [reply]
Hi, yeah sure. Send me the address

**Comment by hjkoster400d** [comment]
Toppie, ik wandel vanmiddag tijdens de lunch even langs een brievenbus.
Success alvast met solderen en wifi bediening van de jacuzzi :D

Gr Rik

Op ma 11 jul. 2022 om 09:04 schreef Deefjes ***@***.***>:

**Comment by Deefjes** [comment]
Hi Remi,

Yes I’ve got one spare board. But to be clear; it’s the V1 board version PCB. Not the V2.

If you send me you’re address info I’ll get at sending it asap.


Met vriendelijke groet,
Dave Leijsten

Op 19 jun. 2023 om 21:32 heeft Remigiusz2020 ***@***.***> het volgende geschreven:

﻿

HI,
Do you have still one left?

BR
Remi

**Comment by Remigiusz2020** [comment]
oh thank you for replay. I need v2 :)    
            
          
     
      
       
        Dnia 20 czerwca 2023 00:18 Deefjes ***@***.***&gt; napisał(a):
       
    
       
         
Hi Remi,
  

  
Yes I’ve got one spare board. But to be clear; it’s the V1 board version PCB. Not the V2.
  

  
If you send me you’re address info I’ll get at sending it asap.
  

  

  
Met vriendelijke groet,
  
Dave Leijsten
  

  
Op 19 jun. 2023 om 21:32 heeft Remigiusz2020 ***@***.***&gt; het volgende geschreven:
  

  
﻿
  

  
HI,
  
Do you have still one left?
  

  
BR
  
Remi
